src/main/java/io/rcktapp/api/handler/dynamo/DynamoDb.java

```diff
@@ -100,6 +100,11 @@ public class DynamoDb extends Db
 
    private AmazonDynamoDB dynamoClient = null;
 
+
+   public DynamoDb() {
+        super();
+        setType("dynamo");
+   }
```

Comment:
By setting type, the log messages on startup are a bit more expressive. (no  ' - null')